chd_builder.go

```diff
@@ -41,13 +41,21 @@ func (b bucketVector) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
 type CHDBuilder struct {
 	keys   [][]byte
 	values [][]byte
+	seed   int64
+	seeded bool
 }
 
 // Create a new CHD hash table builder.
 func Builder() *CHDBuilder {
 	return &CHDBuilder{}
```

Comment:
I think this needs to be updated to set seed to a random value?

Reply:
Given that the constructor just allocates an empty struct, I wasn't sure if everyone used the constructor or whether some users just allocate themselves. 

So I went with an extra boolean and did the seeding when needed rather than up front.

Reply:
That's a better idea than my one, thanks!